extensions/who-is-visiting-this-content/app.R

```diff
@@ -149,7 +202,12 @@ server <- function(input, output, session) {
   )
   output$aggregated_visits <- renderTable(
     aggregated_visits_data() |>
-      rename("Total Visits" = n_visits, "Full Name" = full_name, "Username" = username)
+      rename(
+        "Total Visits" = n_visits,
+        "Number of Hits" = n_hits,
```

Comment:
Is this latest code deployed? I am seeing `n_hits` as a column name in the Aggregated Visits tab.

Reply:
I'm not sure -- will check shortly -- but this code is all gonna change anyway in the next issue where we switch from `DT` to `gt` 😅

Reply:
I just re-deployed the content, so the correct column names should be showing now.